scalameta/semanticdb-scalac-core/src/main/scala/scala/meta/interactive/InteractiveSemanticdb.scala

```diff
@@ -9,7 +9,7 @@ import scala.tools.nsc.Settings
 import scala.tools.nsc.interactive.Global
 import scala.tools.nsc.interactive.Response
 import scala.tools.nsc.reporters.StoreReporter
-import org.langmeta.semanticdb.Document
```

Comment:
FYI I reference org.langmeta in my code because IJ autoimports those modules when I type `Documen<COMPLETE>`. "Go to definition" on the scala.meta type aliases is also quite annoying.